tests/exclude_filter/test

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/perl
+
+use strict;
+
+use Test;
+BEGIN { plan tests => 21 }
+
+use File::Temp qw/ tempfile /;
+use Time::HiRes qw(usleep);
+
+###
+# functions
+
+sub key_gen {
+	my @chars = ("A".."Z", "a".."z");
+	my $key = "testsuite-" . time . "-";
+	$key .= $chars[rand @chars] for 1..8;
+	return $key;
+}
+
+###
+# setup
+
+# reset audit
+system("auditctl -D >& /dev/null");
+
+# create stdout/stderr sinks
+(my $fh_out, my $stdout) = tempfile(TEMPLATE => '/tmp/audit-testsuite-out-XXXX',
+				    UNLINK => 1);
+(my $fh_out2, my $stdout2) = tempfile(TEMPLATE => '/tmp/audit-testsuite-out2-XXXX',
+				    UNLINK => 1);
+
+###
+# tests
+
+my $result;
+my $key = key_gen();
+my $msgtype = "SYSCALL";
+my $pid = $$;
+my $uid = 0;
+my $gid = 0;
+my $auid = 0;
+my $subj_user = "unconfined_u";
+my $subj_role = "unconfined_r";
+my $subj_type = "unconfined_t";
+my $subj_sen = "s0";
+my $subj_clr = "s0:c0.c1023";
```

Comment:
Since we are not running the tests below with a specific SELinux label, we should be set these values at runtime instead of hard coding them in the test.  You should be able to get everything you need from the 'id -Z' command:

    # id -Z
    unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023